siege_utilities/geo/gazetteers/census_gazetteer.py

```
@@ -0,0 +1,321 @@
+            )
+        return shapely_shape(features[0]["geometry"])
+
+    def _row_to_candidate(self, row: Mapping[str, Any]) -> GazetteerCandidate:
+        admin: dict[str, str] = {}
+        if row.get("state_fips"):
+            admin["state_fips"] = str(row["state_fips"])
+        if row.get("county_fips"):
+            admin["county_fips"] = str(row["county_fips"])
+        path: tuple[str, ...] = ("US",)
+        if row.get("state_fips"):
+            path = path + (str(row["state_fips"]),)
```

Ruff (0.15.12)

[warning] Consider `(*path, str(row["state_fips"]))` instead of concatenation

Replace with `(*path, str(row["state_fips"]))`

(RUF005)